lib/src/credential_component.dart

```diff
@@ -1,17 +1,15 @@
 import 'package:angular/angular.dart';
-import 'package:angular_components/material_button/material_button.dart';
 
 import 'credential_service.dart';
 
 @Component(
   selector: 'credential',
   template: '''
-    <material-button (click)="onClick()" debugId="sign-on-btn">
```

Comment:
Why don't we want to use material-button? It's the button component we use over the much older <button> component.

Reply:
The reason for this change is documented here #23. Using button is a temporary solution for now so that we can compile, we are still looking for better options though.